app/web/src/newhotness/Workspace.vue

```diff
@@ -496,12 +496,25 @@ const lobby = computed(
     loadedOnboardingStateFromApi.value === false, // Don't continue before we got the response from the auth API
 );
 
+const componentsOnHead = ref(false);
+const componentsOnHeadApi = useApi(ctx.value);
+const componentsOnHeadData = async () => {
+  const call = componentsOnHeadApi.endpoint<{ componentsOnHead: boolean }>(
+    routes.ComponentsOnHead,
+  );
+  const response = await call.get();
+
+  componentsOnHead.value = response.data.componentsOnHead;
+};
+
 const showOnboarding = computed(() => {
   // Force Onboarding2 if this debug boolean is set to true
   if (DEBUG_MODE) return true;
 
   if (!featureFlagsStore.INITIALIZER_ONBOARD) return false;
 
+  if (componentsOnHead.value === true) return false;
```

Comment:
If we're at all concerned with this endpoint not returning (token failure, SDF down / deploying, etc) we should wrap this with tan stack and use a "undefined" / true / false pattern, so in the case of failure (we know we tried the endpoint) we can let "undefined" skip the onboarding.

Reply:
I can check the status of the endpoint if that would be better?

Reply:
I will follow on with changes to make this better